tests/results/result_actions/OpenEditorActionTest.py

```diff
@@ -216,6 +217,22 @@ def test_open_files_at_position_no_position(self):
                 stdout=subprocess.PIPE
             )
 
+    def test_unknown_editor_warning(self):
+        logger = logging.getLogger()
+        uut = OpenEditorAction()
+        result_mock = Result.from_values(
+            'test', '', self.fa, line=None, column=None,
+        )
+        with unittest.mock.patch('subprocess.call'):
```

Comment:
why this mock if you aren't checking if it had been called? to prevent a subprocess from being created?